lib/translations.json

```diff
@@ -542,38 +542,38 @@
       "key": "metaData/field/emailSubject",
       "value": "E-Mail-Betreff"
     },
-    {
-      "key": "metaData/field/facebookGenerated",
-      "value": "Sharetafel bauen"
-    },
     {
       "key": "metaData/field/facebookTitle",
-      "value": "Titel"
+      "value": "Facebook Titel"
     },
     {
       "key": "metaData/field/facebookDescription",
-      "value": "Deskription"
+      "value": "Facebook Deskription"
```

Comment:
"Deskription" is such a non-word (in German), it's quite funny. I'd change it everywhere to "Lead" (which is what it actually is -> see slack thread with production)